src/garage/torch/q_functions/discrete_cnn_q_function.py

```diff
@@ -73,8 +73,8 @@ def __init__(self,
                  strides,
                  minibatch_size,
```

Comment:
this is a property of the optimizer and not the model -- why is this needed?

Reply:
Oh I initially put that there because `CNNModule` needs to be given the input shape, but it actually just ignores the batch dimension so this turned out to be unnecessary. Fixed.